ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/GroupByParamsFactoryTest.java

```diff
@@ -238,37 +341,84 @@ public void shouldLogProcessingErrorIfAnyMultiExpressionResolvesToNull() {
   }
 
   @Test
-  public void shouldUseNullInGroupByIfOneExpressionFailsOrReturnsNullInMulti() {
```

Comment:
This test is identical to the existing `shouldReturnNullIfAnyMultiExpressionResolvesToNull()` and is therefore redundant.